repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/services/httpconnectionpool/HttpClientStateTest.groovy

```diff
@@ -139,13 +136,12 @@ class HttpClientStateTest extends ReposeValveTest {
             def value = session.getAttribute("KEY")
             println("Session ID: ${session.getId()} HEADER COUNT: ${req.getHeader("x-count-thingy")} KEY:${value} cookies: ${cookies}")
             session.setAttribute("KEY", "VALUE")
-            def cookie = new Cookie("TESTCOOKIE", "TEH VALUE")
+            def cookie = new Cookie("TESTCOOKIE", "THE-VALUE")
```

Comment:
Between jetty [v9.4.2.v20170220 and v9.4.3.v20170317](https://github.com/eclipse/jetty.project/compare/jetty-9.4.2.v20170220...jetty-9.4.3.v20170317) the Cookie value syntax check was updated. The Cookie values are now properly checked against [RFC6265](https://tools.ietf.org/html/rfc6265#section-4.1.1) and may not contain the US-ASCII space character (0x20) which the failing test used.
 - https://github.com/eclipse/jetty.project/commit/528c1f7745fbd20de06d91263c794cd4035c8991